### ISSUE TITLE
When a consumer adds any topic,consumer don't want any previous offset messages ,how it can be achieved ?

### ISSUE THREAD

**Issue by mindfireon812** [SOHU-Co/kafka-node#203]

**Comment by haio** [comment]
If you are using Consumer, you can set specific offset to the topic and set fromOffset to true, then consumer will get message from the given offset, for example:
```
var fromOffset = true
consumer.addTopics([{ topic: 't', offset:10 }], cb, fromOffset)
```
HighLevelConsumer#addTopic seems has bug, don't use it at the moment.

**Comment by shiyeling** [comment]
hi haio,
Any plan to fix this bug with HighLevelConsume#addTopics method? Need it now in a project but it's broken, unfortunately.

**Comment by chrishiestand** [comment]
Yes, this seems to be broken. We should file a new issue for this problem or rename this one.
